scripts/_setup-macos.sh

```diff
@@ -35,7 +35,8 @@ set -x # Print commands that are executed.
 FB_OS_VERSION=v2021.05.10.00
 NPROC=$(sysctl -n hw.physicalcpu)
 COMPILER_FLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt"
-DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
+DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}/_build
```

Comment:
Wondering if it makes sense to do something like 
```
DEPENDENCY_DIR=XXX 
```
 
in `build_mac_dep.sh` ? -- so we can keep this file sync with the Velox repo on ?

Reply:
yeah, doable.